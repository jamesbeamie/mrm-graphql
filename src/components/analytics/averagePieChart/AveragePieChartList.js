import React from 'react';
import '../../../../src/assets/styles/averagePieChartList.scss';
import BookingsCountBarGraph from '../barGraph/BookingsCountBarGraph';
import AverageMeetingDurationPieCharts from './AverageMeetingDurationPieChart';
import GetAverageRoomCapacity from './GetAverageRoomCapacity';

const AveragePieChartList = () => (
  <div className="pie-chart-container">
    <AverageMeetingDurationPieCharts />
    <GetAverageRoomCapacity />
    <BookingsCountBarGraph />
  </div>
);

export default AveragePieChartList;
