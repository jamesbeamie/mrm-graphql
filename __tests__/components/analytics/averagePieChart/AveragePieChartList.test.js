import React from 'react';
import { shallow } from 'enzyme';
import AveragePieChartList from '../../../../src/components/analytics/averagePieChart/AveragePieChartList';

describe('Average Pie Chart Component', () => {
  it('renders correctly from memory', () => {
    expect(shallow(<AveragePieChartList />)).toMatchSnapshot();
  });
});
