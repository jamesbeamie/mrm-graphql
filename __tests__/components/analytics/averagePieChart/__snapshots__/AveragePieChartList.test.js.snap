// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Average Pie Chart Component renders correctly from memory 1`] = `
<div
  className="pie-chart-container"
>
  <AverageMeetingDurationPieChart />
  <GetAverageRoomCapacityComponent />
  <BookingsCountBarGraph />
</div>
`;
